examples/step-51/doc/builds-on

```diff
@@ -1 +1 @@
-step-7 step-9
+step-7 step-9 step-61
```

Comment:
Is there a particular reason why you added a dependency on `step-61` here? `step-61` already depends on `step-51`. At least I would find it confusing that the dependency graph implies a circular dependency between the two steps. As you write further down, it's two related ideas executed with different finite element spaces (and mathematical tools, in part).

Reply:
The mutual dependency will make sure that they appear at the same level in the graph, which is what I think they should. None of them *builds* on the other apart from referencing ideas.